tooling/eslint-config/base.js

```diff
@@ -39,6 +39,15 @@ const config = {
     ],
     "import/consistent-type-specifier-style": ["error", "prefer-top-level"],
   },
+  overrides: [
+    {
+      files: ["**/test/**", "**/*.test.ts", "**/*.test.tsx"],
+      extends: ["plugin:@typescript-eslint/disable-type-checked"],
+      rules: {
+        "@uploadthing/no-throwing-promises": "off",
+      },
+    },
+  ],
```

Comment:
bunch of the expect utilis returns any and it's just annoying having to disable rules everywhere in tests...